src/Service.GraphQLBuilder/Directives/RelationshipDirective.cs

```diff
@@ -54,11 +54,8 @@ public static string Target(FieldDefinitionNode field)
         /// <returns>The name of the target object if the relationship is found, null otherwise.</returns>
         public static string? GetTarget(IInputField infield)
         {
-            Directive? directive = (Directive?)infield.Directives.FirstOrDefault(d => d.Name.Value == DirectiveName);
-            DirectiveNode? directiveNode = directive?.ToNode();
-            ArgumentNode? arg = directiveNode?.Arguments.First(a => a.Name.Value == "target");
-
-            return (string?)arg?.Value.Value;
+            Directive? directive = (Directive?)infield.Directives.FirstOrDefault(DirectiveName);
```

Comment:
The directive collection has now optimized lookup methods. which reduces the code needed to get things from type system members.